### ISSUE TITLE
It's not possible to configure code coverage whitelist without configuration file

### ISSUE THREAD

**Issue by SvetlanaZem** [sebastianbergmann/phpunit#1872]
In latest PHPUnit 5.0 version (25.09.15) there's is an option "whitelist" that allow you to add directory for code coverage, but it won't be used if you don't have whitelist configured in configuration file.

See:
TestRunner.php:325 
```php
if (!$this->codeCoverageFilter->hasWhitelist()) {
    $this->writeMessage('Error', 'No whitelist configured, no code coverage will be generated');

   $codeCoverageReports = 0;
}
```

TestRunner.php:366
If whitelist is not configured in configuration file $codeCoverageReports will be 0, due to previous code fragment.
```php
if ($codeCoverageReports > 0) { 
    // ..... 
    if (isset($arguments['whitelist'])) {
        $this->codeCoverageFilter->addDirectoryToWhitelist($arguments['whitelist']);
    }

    $result->setCodeCoverage($codeCoverage);
}
```

**Comment by sebastianbergmann** [comment]
Thank you, Svetlana, for reporting this issue.

**Comment by alberto56** [comment]
This seems to have been fixed after 5.0, so this problem persists in 5.0. I have tried adding the following to my phpunit.xml file, but the error still persists.

    <filter>
      
      <whitelist processUncoveredFilesFromWhitelist="true">
      </whitelist>
    </filter>

However I'm still getting this issue. If anyone knows the simplest addition I can make to the xml file in order to not get this error with 5.0, please share.

**Comment by sebastianbergmann** [comment]
This issue has been fixed before PHPUnit 5.0.0 was released, @alberto56. Please open a new ticket and explain the issue that you are experiencing.

**Comment by alberto56** [comment]
Thanks @sebastianbergmann, I have opened a new issue at #1932

Cheers,

Albert.
